resources/api/resource.py

```diff
@@ -317,9 +320,49 @@ def filter_is_favorite(self, queryset, name, value):
         else:
             return queryset.exclude(favorited_by=self.user)
 
+    def _deserialize_datetime(self, value):
+        try:
+            return arrow.get(value).datetime
+        except ParserError:
+            raise exceptions.ParseError("'%s' must be a timestamp in ISO 8601 format" % value)
+
+    def filter_available_between(self, queryset, name, value):
+        if len(value) != 2:
+            raise exceptions.ParseError('available_between takes exactly two comma-separated values.')
+
+        available_start = self._deserialize_datetime(value[0])
+        available_end = self._deserialize_datetime(value[1])
+
+        if available_start.day != available_end.day:
```

Comment:
You probably want to compare the `date`s, not the `day`s. (`2016-01-01T00:00:00,2044-10-01T00:00:00`)